.coafile

```diff
@@ -0,0 +1,27 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js, tmp/**,
```

Comment:
why is `.eslintrc.js` ignored globally?

Reply:
Yes, it raised warning as eslint itself won't lint .eslintrc.js. I dont know how to remove the warning. And force lint it, so ignore. 
Should be ignored in js, instead of global.

Reply:
*eslintbear. Not eslint itself

Reply:
```
Traceback (most recent call last):
  File "/Users/bekicot/Library/Python/3.6/lib/python/site-packages/coalib/bears/Bear.py", line 282, in execute
    return [] if result is None else list(result)
  File "/usr/local/lib/python3.6/site-packages/bears/js/ESLintBear.py", line 88, in process_output
    if result['ruleId'] is not None else self)
KeyError: 'ruleId'
```

Reply:
No suficient knowledge to debug it in timely maner

Reply:
I also try

```console
./node_modules/eslint --no-ignore --no-color --stdin --stdin-filename=/home/travis/build/coala/git-task-list/.eslintrc.js --config .eslintrc.js < .eslintrc.
```

it works fine.

Reply:
GCI Leader seems to have the same problem.

Reply:
`coala --files=git-task-list/.eslintrc.js --bears=ESLintBear -V`

Reply:
I asked you (two comments earlier right here) to push to travis and show me a travis error.
Anyway ... I did it, and now I can see it
https://travis-ci.org/jayvdb/gsoc-prep-tasks/jobs/391075675#L958

I see you did create an issue https://github.com/coala/coala-bears/issues/2533

Now add a comment in your .coafile about that bug.

Reply:
It is not raised an error, just a warning. It was anoying so i ignored it